packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -32,6 +32,18 @@ Given("I'm running the docs page {string}", (page: string) => {
   });
 });
 
+Given("I'm running the docs page", () => {
+  cy.visit('/', {
+    // See: https://glebbahmutov.com/blog/cypress-tips-and-tricks/#control-navigatorlanguage
+    onBeforeLoad(win) {
+      Object.defineProperty(win.navigator, 'language', { value: language });
```

Comment:
For my own learning, what is this for?

Reply:
This is from https://github.com/aws-amplify/amplify-ui/pull/408/files#r713447746, for i18n.

To be honest I just copied and pasted this, and this is unneeded in docs. I'll remove it!

Reply:
Done in 84dbc2e1b52b83c74c48e267fce0de30a9393759